adev/shared-docs/pipeline/api-gen/extraction/extract_api_to_json.bzl

```
@@ -77,6 +80,9 @@ extract_api_to_json = rule(
             mandatory = True,
             allow_single_file = True,
         ),
+        "private_modules": attr.string_list(
+            doc = """List of private modules that should not be included in the API reference""",
```

Maybe something like?

```suggestion
            doc = """List of private modules that should not be included in the API symbol linking""",
```